spe.py

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/env python
+'''
+Do the Single Photoelectron anaylsis
+
+Usage:
+  spe.py [options] <files>...
+
+Options:
+  -h --help     Show this screen.
+'''
+
+from docopt import docopt
+from digicampipe.io.event_stream import event_stream
+import numpy as np
+import matplotlib.pyplot as plt
+from histogram.histogram import Histogram1D
```

Comment:
I think this "histogram.histogram" does not exist.

Reply:
correct: at the moment I have it here : https://github.com/calispac/histogram

just a class to create histograms on an "event" basis. I tried to keep it general (i.e. not only for digicampipe) and fast : 

have a look at the example : 
https://github.com/calispac/histogram/blob/master/example.py

For the moment I will add this as a required external packages.

Reply:
I wrote something similar recently .. maybe we can combine or so.

Reply:
yes that would be great!